pkgs/top-level/all-packages.nix

```diff
@@ -26838,14 +26838,18 @@ in
   };
 
   kodi-wayland = callPackage ../applications/video/kodi {
+    jre_headless = jdk11_headless;
     waylandSupport = true;
   };
 
   kodi-gbm = callPackage ../applications/video/kodi {
+    jre_headless = jdk11_headless;
     gbmSupport = true;
   };
 
-  kodi-cli = callPackage ../tools/misc/kodi-cli { };
+  kodi-cli = callPackage ../tools/misc/kodi-cli {
+    jre_headless = jdk11_headless;
+  };
```

Comment:
`kodi-cli` isn't a variant of `kodi`, it is just a shell script or something. It doesn't take `java` at all so you can remove this.

Reply:
I first tested building it, and it failed, but actually it was a dependency that failed to build. Fixed it now.